package.json

```diff
@@ -31,5 +34,8 @@
     "stylelint": "^12.0.0",
     "stylelint-config-standard": "^19.0.0",
     "typescript": "^4.6.2"
+  },
+  "dependencies": {
+    "walk-sync": "^2.0.2"
```

Comment:
> In order for ESLint to not throw lint errors about the imports in clean.js, I added walk-sync to the dependency of root package.json. This is the first package that does that, so I want to make sure that we are okay in this.

I think we should aim to preserve the original function of the eslint rule (avoid importing dev dependencies in user-facing code) here by using line-by-line `eslint-ignore`. It forces conscious consideration of whether this dependency is really being used for "dev-only" purposes at a small cost (1 extra comment)

Reply:
Agree with you on keeping the rule as-is. I was inclined to put the package in `devDependencies`, but that would result in the lint error saying that it should be in `dependencies` instead (it doesn't know that this is just a dev script), so I just followed the suggestion.

Reading your comment, do you mean that it is advisable to just put the `eslint-ignore` instead of putting it in `dependencies` in this case?

Reply:
Yup.

> should be in dependencies instead (it doesn't know that this is just a dev script), so I just followed the suggestion.

The other risk with this (specific to this unpublished root package.json in the monorepo) is that it becomes possible to import `walk-sync` (which is not published) from irrelevant subpackages like vue-components. Though I can't recall if there's an eslint rule to detect this case / we have it enabled =P